package-lock.json

```diff
@@ -191,6 +191,47 @@
         }
       }
     },
+    "@brightcove/kacl": {
+      "version": "0.1.8",
+      "resolved": "https://a0us.jfrog.io/a0us/api/npm/npm/@brightcove/kacl/-/kacl-0.1.8.tgz",
```

Comment:
Are we supposed to avoid private npm registers in open-source projects like this?

Reply:
No there shouldn't be, seems if you have an .npmrc in your home directory, `npm i` will attempt to use that registry, although none of our dependencies are coming from that registry.